pkg/daemon/ceph/osd/remove.go

```diff
@@ -138,5 +146,29 @@ func removeOSD(clusterdContext *clusterd.Context, clusterInfo *client.ClusterInf
 		logger.Errorf("failed to remove CRUSH host %q. %v", hostName, err)
 	}
 
+	// The warning should be silenced by archiving the crash
```

Comment:
What warning are we silencing? A ceph health warning?

Reply:
yes

Reply:
How about adding more details to the comment?